hadoop-ozone/s3-secret-store/src/test/java/org/apache/hadoop/ozone/s3/remote/vault/VaultS3SecretStoreTest.java

```diff
@@ -48,7 +46,10 @@
  */
 public class VaultS3SecretStoreTest {
```

Comment:
This reminds me: please rename this test class to `TestVaultS3SecretStore` to match naming pattern executed in test runs.

Reply:
On second thought, there are other, unrelated tests with the same problem, so let's address this separately.